eth_account/hdaccount/mnemonic.py

```diff
@@ -56,7 +56,7 @@
 _cached_wordlists: Dict[str, List[str]] = dict()
 
 
-def get_wordlist(language):
+def get_wordlist(language: str) -> List[str]:
```

Comment:
I wonder if we should make language an enum instead of accepting any `str`?

Reply:
I think that's a good idea. OK with me doing that in a separate issue/pr?